issuer-front-end/src/CreateSchemaAndMetadataFiles.tsx

```diff
@@ -274,56 +284,56 @@ export default function CreateSchemaAndMetadataFiles() {
     return (
         <>
             <TestBox>
-                <div
-                    className="containerToolTip"
-                    role="presentation"
-                    onClick={display}
-                    data-toggle="tooltip"
-                    title="The credentialSchema is a JSON object that you will create in this step. You should host this JSON object somewhere publicly on the web so it is available via a URL. You will use the URL in step 4. You can for example host it on your gist account (`https://gist.github.com/`) and click the `raw` button to optain the URL (e.g. `https://gist.githubusercontent.com/DOBEN/bfe30ecea16f7a3ea1b87aa40902b9ac/raw/a8ab51fca489d04710fb19fb7122bb283dba719a/gistfile1.txt`)."
-                >
-                    <div>
-                        <h3>CredentialSchema</h3>
-                    </div>
-                    <div className="infolink" />
+                <div>
+                    <h3>CredentialSchema</h3>
+                    <p>
+                        {' '}
+                        The <strong> credentialSchema </strong> is a JSON schema describing the credential. The schema
+                        must be hosted at a public URL so that it is accessible to the wallet, which uses it, among
+                        other things, to render credentials.{' '}
+                    </p>
+
+                    <p>
+                        {' '}
+                        The schema consists of some metadata (name of the credential and description) together with a
+                        number of attributes. The form below supports inputting the necessary data and generating the
+                        JSON schema in the correct format.{' '}
+                    </p>
                 </div>
                 <br />
+                <label>
+                    {' '}
+                    Credential name
```

Comment:
Currently, the `credential name` and the `credential description` look slightly different compared to the other input field texts. Suggestion to keep it consistent or change the other input fields as well.